addons/io_hubs_addon/components/definitions/particle_emitter.py

```diff
@@ -117,6 +120,25 @@ def gather(self, export_settings, object):
         }
         return props
 
+    def migrate(self, migration_type, panel_type, instance_version, host, migration_report, ob=None):
+        migration_occurred = False
+        if instance_version < (1, 1, 0):
+            migration_occurred = True
+            startVelocity = self.startVelocity.copy()
+            startVelocity = Vector((startVelocity.x, startVelocity.z, startVelocity.y))
+            self.startVelocity = startVelocity
+
+            endVelocity = self.endVelocity.copy()
+            endVelocity = Vector((endVelocity.x, endVelocity.z, endVelocity.y))
+            self.endVelocity = endVelocity
+
+            if migration_type != MigrationType.GLOBAL or is_linked(ob) or type(ob) == bpy.types.Armature:
+                host_reference = get_host_reference_message(panel_type, host, ob=ob)
+                migration_report.append(
+                    f"Warning: The Particle Emitter component's Y and Z on startVelocity and endVelocity on the {panel_type.value} {host_reference} may not have migrated correctly")
```

Comment:
I don't think a special warning needs to be displayed for linking/appending since this is just a normal version bump, although it's possible it may be needed on armatures that don't have an associated object, but I don't know.